utils/frame/remote-externalities/src/lib.rs

```diff
@@ -1087,88 +1138,19 @@ mod tests {
 #[cfg(all(test, feature = "remote-test"))]
 mod remote_tests {
 	use super::test_prelude::*;
-	use std::os::unix::fs::MetadataExt;
```

Comment:
why all of these code removals? I think you branched off form the wrong branch. Start from the latest code in my base branch, and merge master please.